### PR TITLE
Add humble to the supported distros

### PR DESCRIPTION
**Pull request by ipa-cmh**
We should add humble to the supported distros. 
I think this will do it but I still need to test.

### CODE REVIEW

**Comment by Briancbn** [comment]
Add in the test here below too?
https://github.com/ros-industrial/industrial_ci/blob/25481e5f15476812bde1a223765f602eb07eaa2d/.github/workflows/main.yml#L67

**Comment by ipa-cmh** [comment]
@Briancbn this is working. Can you merge?

**Comment by Briancbn** [comment]
@mathias-luedtke please help take a look:)

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -64,6 +64,7 @@ jobs:
           - {ROS_DISTRO: noetic, AFTER_SCRIPT: 'rosenv rosrun industrial_ci run_travis', ADDITIONAL_DEBS: "ros-noetic-rosbash"}
           - {ROS_DISTRO: foxy, AFTER_SCRIPT: 'rosenv ros2 run industrial_ci run_travis', ADDITIONAL_DEBS: "ros-foxy-ros2run"}
           - {ROS_DISTRO: galactic, TRACE: true}
+          - {ROS_DISTRO: humble, ROS_REPO: testing}
```
Comment (mathias-luedtke) [inline .github/workflows/main.yml:67]: Please do not set the ROS_REPO here.
testing is the default anyway!

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -238,6 +239,10 @@ jobs:
           - repo: 'ros-controls/control_msgs'
             ref: 'galactic-devel'
             env: {ROS_DISTRO: galactic, PRERELEASE: true}
+          
+          - repo: 'ros-controls/control_msgs'
+            ref: 'galactic-devel'
+            env: {ROS_DISTRO: humble, PRERELEASE: true, ROS_REPO: testing}
```
Comment (mathias-luedtke) [inline .github/workflows/main.yml:245]: ROS_REPO is ignores for pre-releases

**Review by mathias-luedtke — CHANGES_REQUESTED** [review]
Thank you very much!
I was planning to add this for some time, but I haven't had time yet..
Please remove the `ROS_REPO` setting and it will be ready for merge :)

**Comment by mathias-luedtke** [comment]
And please add it in the documentation as well: https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#supported-ros-distributions

File: .github/workflows/main.yml (hunk 1 above)
Comment (ipa-cmh) [reply .github/workflows/main.yml]: Done

File: .github/workflows/main.yml (hunk 2 above)
Comment (ipa-cmh) [reply .github/workflows/main.yml]: Done

**Review by mathias-luedtke — APPROVED** [review]

**Pull request merged**